src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.html

```diff
@@ -1,22 +1,40 @@
-<ng-container *ngIf="subVolumes$ | async as subVolumes">
-  <cd-table [data]="subVolumes"
-            columnMode="flex"
-            [columns]="columns"
-            selectionType="single"
-            [hasDetails]="false"
-            (fetchData)="fetchData()"
-            (updateSelection)="updateSelection($event)">
+<div class="row">
+  <div class="col-sm-2">
+    <h2 i18n>Groups</h2>
+    <ng-container *ngIf="subVolumeGroups$ | async as subVolumeGroups">
+      <button class="btn btn-primary m-1"
+              type="button"
+              [class.active]="activateButtonIndex === 0"
+              (click)="selectSubVolumeGroup('', 0)"
+              autofocus>Default</button>
+      <div *ngFor="let subVolumeGroup of subVolumeGroups; let i = index">
+        <button class="btn btn-primary m-1"
+                type="button"
+                [class.active]="activateButtonIndex === i + 1"
+                (click)="selectSubVolumeGroup(subVolumeGroup.name, i + 1)">{{subVolumeGroup.name}}</button>
+      </div>
```

Comment:
instead of buttons, we can use the bootstrap [nav-tabs](https://getbootstrap.com/docs/5.0/components/navs-tabs/#vertical).

```
<ul class="nav flex-column">
    <li class="nav-item"
        *ngFor="let subVolumeGroup of subVolumeGroups; let i = index">
      <a class="nav-link" [ngClass]="{active: acivateButtonIndex === i+1"> 
      ......
      ....
      </ul>
 ```

Reply:
instead of sending and storing the index why not store the `activeGroup` name and compare it with the current name?